backend/sozisel/lib/sozisel/model/utils.ex

```diff
@@ -9,4 +9,30 @@ defmodule Sozisel.Model.Utils do
     |> String.replace("%", "\\%")
     |> String.replace("_", "\\_")
   end
+
+  def from_deep_struct(%{} = map), do: convert(map)
```

Comment:
Just a question, did you come up with it yourself? 

I would also add `@doc` for that explaining what it does and why we need this.